build-utils/mount-partitions.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
```

Comment:
This is not really a build-util, it's a runtime util. We should create a new folder or just put it on the same level as the `entry.sh` script

Reply:
Moved to project root in 71cddb6a1b9d51a59a350726a80035e5afce1403